docs/platforms/python/performance/instrumentation/cache-module.mdx

```diff
@@ -0,0 +1,100 @@
+
+You need to create a span when you put something into the cache and create a second span when you fetch something out of the cache. By adding some additional information to those cache Sentry will show you how your caching system is performing.
+
+Always make sure that there is a transaction running when you create the spans. If you are using a web framework then those transactions are created for you. See <PlatformLink to="/performance/">Performance Monitoring</PlatformLink> for more information.
+
+For detailed information about data that can be set, see the [Cache Module Developer Specification](https://develop.sentry.dev/sdk/performance/modules/cache/).
+
+## Manual Instrumentation
+
+If you use <PlatformLink to="/integrations/django/">Django</PlatformLink>, <PlatformLink to="/integrations/redis/">Redis</PlatformLink>,  memcached *(comming soon)* then you do not need manual instrumentation. See the linked docs on how to enable automatic cache instrumentation.
+
+If you have caching solution not mentioned above you need to do the following:
+
+### Initialize Sentry
+
+<PlatformContent includePath="getting-started-config" />
+
+### Add Span When Putting Data Into the Cache:
+
+If the cache you’re using isn’t supported by Opt-In instrumentation mentioned above, you can use the <PlatformLink to="/performance/instrumentation/custom-instrumentation/">Custom Instrumentation</PlatformLink> instructions below to emit cache spans:
+
+1. Set the cache value with whatever cache library you happen to be using.
+2. Wrap the part of your application that uses the cached value with  `with sentry_sdk.start_span(...)`
+3. Set `op` to `cache.set`.
+4. Set `cache.item_size` to an integer representing the size of the cached item.
+
+(The steps described above are documented in the snippet.)
+
+```python
+import my_caching
+import sentry_sdk
+
+key = "myCacheKey123"
+value = "The value I want to cache."
+
+with sentry_sdk.start_span(op="cache.set") as span:
+    # Set a key in your caching using your custom caching solution
+    my_caching.set(key, value)
+
+    # Describe the cache server you are accessing
+    span.set_data("network.peer.address", "cache.example.com/supercache")
+    span.set_data("network.peer.port", 9000)
+
+    # Add the key you want to set
+    span.set_data("cache.key", key)
+
+    # Add the size of the value you stored in the cache
+    span.set_data("cache.item_size", len(value))  # Warning: if value is very big this could use lots of memory
+```
+
+
+### Add Span When Retrieving Data From the Cache:
+
+If the cache you’re using isn’t supported by Opt-In instrumentation mentioned above, you can use the <PlatformLink to="/performance/instrumentation/custom-instrumentation/">Custom Instrumentation</PlatformLink> instructions below to emit cache spans:
```

Comment:
```suggestion
If the cache you’re using isn’t supported by opt-In instrumentation mentioned above, you can use the <PlatformLink to="/performance/instrumentation/custom-instrumentation/">Custom Instrumentation</PlatformLink> instructions below to emit cache spans:
```
Same two questions as above :)

Reply:
You are right. I wanted to refer to the auto instrumentation mentioned above. And yes, linking out is not necessary!